src/styles/variables.css

```diff
@@ -46,6 +46,7 @@
   --temp-grey-blue-6: rgb(231, 242, 246); /* Used for user profile gradient end color */
   --temp-grey-blue-7: rgb(118, 116, 139); /* Used for the wallet connect button in the user navigation */
   --temp-grey-blue-8: rgb(238, 242, 245); /* Used for the actions list hover background */
+  --temp-grey-blue-9: rgb(200, 214, 245); /* Used for Safe Control Checkbox */
```

Comment:
Let's remove this and instead use `--temp-grey-blue-4` which matches it pretty well.